### PR TITLE
use append stead loop and fix comment

### PR DESCRIPTION
**Pull request by Jerry-yz**
#### What type of PR is this?
type fix

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
none
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by jiahuif** [comment]
/ok-to-test
to see if the vendor checks pass.

**Comment by jiahuif** [comment]
Could you redirect changes to the yaml package to 
https://github.com/kubernetes-sigs/yaml
Thank you?

**Comment by Jerry-yz** [comment]
> Could you redirect changes to the yaml package to https://github.com/kubernetes-sigs/yaml Thank you?

ok

> Could you redirect changes to the yaml package to https://github.com/kubernetes-sigs/yaml Thank you?

ok, i redirect commit

File: pkg/generated/openapi/zz_generated.openapi.go [hunk 1]
```diff
@@ -59815,7 +59815,7 @@ func schema_k8sio_kube_scheduler_config_v1_KubeSchedulerConfiguration(ref common
...
+							Description: "PercentageOfNodesToScore is the percentage of all nodes that once found feasible for running a pod, the scheduler stops its search for more feasible nodes in the cluster. This helps improve scheduler's performance. Scheduler always tries to find at least \"minFeasibleNodesToFind\" feasible nodes no matter what the value of this flag is. Example: if the cluster size is 500 nodes and the value of this flag is 30, then scheduler stops finding further feasible nodes once it finds 150 feasible ones. When the value is 0, default percentage (5%--50% based on the size of the cluster) of the nodes will be scored. It is overridden by profile level PercentageOfNodesToScore.",
```
Comment (googs1025) [inline pkg/generated/openapi/zz_generated.openapi.go:59818]: This is a generated file and we should not modify it. 
```
// Code generated by openapi-gen. DO NOT EDIT.
```

Comment (googs1025) [reply pkg/generated/openapi/zz_generated.openapi.go:59818]: Where is the typo? [KubeSchedulerConfiguration](https://github.com/kubernetes/kubernetes/blob/7c780186d7a9020b3a738081719d2f9697d91932/staging/src/k8s.io/kube-scheduler/config/v1/types.go#L68)

**Comment by googs1025** [comment]
We should merge the code into one commit @Jerry-yz

**Comment by Jerry-yz** [comment]
> We should merge the code into one commit @Jerry-yz

thanks you review

**Pull request closed without merging**